app-misc/lirc/lirc-0.9.4a-r2.ebuild

```diff
@@ -91,6 +94,10 @@ src_install() {
 	if ! use static-libs; then
 		rm "${D}/usr/$(get_libdir)/liblirc_client.la" || die
```

Comment:
2 .pc files are installed:
* /usr/lib64/pkgconfig/lirc.pc
* /usr/lib64/pkgconfig/lirc-driver.pc

Therefore, I have taken these actions:
* Removed the "static" use flag
* Added `prune_libtool_files --modules` to `src_install`

By the guidelines given at https://blog.flameeyes.eu/2009/09/removing-la-files-for-dum-w-uncertain-people I think I'm addressing this situation correctly.